From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nabeel Sowan <nabeel@vibes.se>
Date: Fri, 18 Sep 2026 00:06:58 +0200
Subject: [PATCH 05/12] put the docked action toggle beside the lid-close
 button

The two docked toggles were grouped together in the Lock Screen section,
but the action one is about the lid-close action, which the dialog
configures under Button Events next to "Lid close Button:". A tester
looking for it went to Button Events first and did not find it, which is
the dialog's own organisation pointing where it belongs.

Move "Perform the lid-close action, even when docked" into the Button
Events group. The lock toggle stays in Lock Screen beside "Lock screen on
lid close"; each docked toggle now sits with the setting it qualifies.

Signed-off-by: Nabeel Sowan <nabeel@vibes.se>
---
 src/configure_Dialog.ui | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/configure_Dialog.ui b/src/configure_Dialog.ui
index 47da624..ec113e1 100644
--- a/src/configure_Dialog.ui
+++ b/src/configure_Dialog.ui
@@ -1649,7 +1649,15 @@
                                         </size>
                                     </property>
                                 </widget>
-                                <spacer row="5" column="2">
+                                <widget class="TQCheckBox" row="5" column="0" rowspan="1" colspan="3">
+                                    <property name="name">
+                                        <cstring>cB_actionLidDocked</cstring>
+                                    </property>
+                                    <property name="text">
+                                        <string>Perform the lid-close action, even when docked</string>
+                                    </property>
+                                </widget>
+                                <spacer row="6" column="2">
                                     <property name="name">
                                         <cstring>spacer89</cstring>
                                     </property>
@@ -1881,14 +1889,6 @@
                                         <string>Lock screen on lid close, even when docked</string>
                                     </property>
                                 </widget>
-                                <widget class="TQCheckBox" row="4" column="0" rowspan="1" colspan="3">
-                                    <property name="name">
-                                        <cstring>cB_actionLidDocked</cstring>
-                                    </property>
-                                    <property name="text">
-                                        <string>Perform the lid-close action, even when docked</string>
-                                    </property>
-                                </widget>
                                 <spacer row="5" column="1">
                                     <property name="name">
                                         <cstring>spacer79</cstring>
